config/config-observability.yaml

```diff
@@ -99,6 +99,11 @@ data:
     logging.revision-url-template: |
       http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))
 
+    # If non-empty, this enables queue proxy writing request logs to stdout.
+    # The value determines the shape of the request logs and it must be a valid go text/template.
```

Comment:
Could you expand this comment ~significantly to include the full set of fields available to the template?

See my comments below about how to handle the environment also.  It won't be clear to folks what env vars are available, so we should expose a small fixed set of those here, and document how to access those.

Reply:
Will do.